packages/web/README.md

```diff
@@ -3,3 +3,72 @@ Natura Design System - Web Components
 
 ![Travis (.org)](https://img.shields.io/travis/natura-cosmeticos/natds.svg)
 [![Known Vulnerabilities](https://snyk.io/test/github/natura-cosmeticos/natds/badge.svg?targetFile=package.json)](https://snyk.io/test/github/natura-cosmeticos/natds?targetFile=package.json)
+
+---
+
+## Install
+
+```
+$ yarn add @naturacosmeticos/natds-web
+```
+
+## Importing
+
+```
+import { Chip } from '@naturacosmeticos/natds-web'
+
+return (
+  <Chip clickable={true} onDelete={this.onDelete} />
+)
+```
+
+## Docs
+
+https://natura-cosmeticos.github.io/natds/
+
+
+---
+
+## Development
```

Comment:
Everything from this line on should be at the end of the `CONTRIBUTING.md` file

Reply:
Instead of `## Development`, you could add somethind like this:
```
## Contributing

To contribute, please check out our contributing file [link-to-CONTRIBUTING.md]
```